code/modules/research/xenoarchaeology/tools/tools_pickaxe_vr.dm

```diff
@@ -0,0 +1,126 @@
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Excavation pickaxes - sorted in order of delicacy. Players will have to choose the right one for each part of excavation.
+
+/obj/item/weapon/pickaxe/brush
```

Comment:
I guess I'm wondering: Why copy/paste the entire file's contents instead of just overidding the one variable you want to be different?